Assets/HighScore.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Linq;
+using System.Text;
+using System.Collections.Generic;
+
+class HighScore : IComparable<HighScore>
```

Comment:
Right now it's just `Score`. It isn't known if it's high or low from the context of this class :)

Reply:
I have no idea then what the name for 'public int Score { get; set; }'. In my opinion leaving this name will not affect anything later.